### PR TITLE
Prepare 0.6.0 release

### PR DESCRIPTION
**Pull request by kvch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+## [0.6.0]
+
 ### Added
```
Comment (ruflin) [inline CHANGELOG.md]: @kvch Could you remove `deprecated` on line 26?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
```
Comment (andrewkroh) [inline CHANGELOG.md]: We should do this in two commits. One without the "unreleased" section and one with. Then we will tag the first commit with v0.6.0.

**Comment by ruflin** [comment]
Seeing this is not merged yet, should we include the most recent changes / fixes in this release?

@kvch @andrewkroh

**Comment by kvch** [comment]
I rebased the branch. Now it includes the all changes since the last release and a new unreleased section.

**Pull request merged**